### PR TITLE
refactor(iplb): remove uib modal ssl certificates

### PR DESCRIPTION
**Pull request by varun257**
remove uib modal in iplb
MANAGER-4099

Signed-off-by: Varun257 <varun257@gmail.com>

### CODE REVIEW

**Comment by frenautvh** [comment]
some conflicts

**Comment by varun257** [comment]
> some conflicts

working on it

**Comment by varun257** [comment]
When i get conflicts because of prettier, can i just do `git rebase -Xtheirs develop` to rebase??

**Comment by antleblanc** [comment]
> When i get conflicts because of prettier, can i just do `git rebase -Xtheirs develop` to rebase??

Sure! You can!

When you've done with the refactoring, please re-run the following command from the root of the monorepo:

```sh
$ yarn run lint:js
```

Thanks!

File: packages/manager/modules/iplb/src/sslCertificate/preview/iplb-ssl-certificate-preview.controller.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
 export default class IpLoadBalancerSslCertificatePreviewCtrl {
   /* @ngInject */
-  constructor($uibModalInstance, ssl) {
-    this.$uibModalInstance = $uibModalInstance;
-    this.ssl = ssl;
-  }
-
   dismiss() {
-    this.$uibModalInstance.dismiss();
+    return this.goBack();
```
Comment (JeremyDec) [inline packages/manager/modules/iplb/src/sslCertificate/preview/iplb-ssl-certificate-preview.controller.js]: Same here, useless controller

File: packages/manager/modules/iplb/src/frontends/delete/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import routing from './iplb-frontends-delete.routing';
+import deleteComponent from './iplb-frontends-delete.component';
+
+const moduleName = 'ovhManagerIplbFrontendsDeleteModule';
```
[the remark this answers is not in the document]
Comment (varun257) [reply packages/manager/modules/iplb/src/frontends/delete/index.js]: I got a comment to do exactly reverse of this in my previous PR :)

File: packages/manager/modules/iplb/src/frontends/delete/iplb-frontends-delete.controller.js [hunk 3]
```diff
@@ -1,38 +1,27 @@
...
-        this.IpLoadBalancerFrontendsService.deleteFrontend(
-          this.type,
-          this.$stateParams.serviceName,
-          this.frontendId,
-        )
-          .then((response) => this.$uibModalInstance.close(response))
-          .catch((error) => this.$uibModalInstance.dismiss(error)),
+      loaderFunction: () => this.IpLoadBalancerFrontendsService
+        .deleteFrontend(this.type, this.serviceName, this.frontendId)
+        .finally(() => this.goBack()),
```
[the remark this answers is not in the document]
Comment (varun257) [reply packages/manager/modules/iplb/src/frontends/delete/iplb-frontends-delete.controller.js]: The success and error messages are handled in the API calls in service itself. This code was not returning anything.

[the remark this answers is not in the document]
Comment (marie-j) [reply packages/manager/modules/iplb/src/frontends/delete/iplb-frontends-delete.controller.js]: There is no message displayed to the user depending on whether the call succeeds/fails ?

[the remark this answers is not in the document]
Comment (varun257) [reply packages/manager/modules/iplb/src/frontends/delete/iplb-frontends-delete.controller.js]: It does, but the messages are handled in the service itself, on API success/failure, there are common messages being pushed to the main route from the service. So its not handled in the controller.

**Comment by antleblanc** [comment]
@varun257 Pull Request has been inactive for a long period so I will close it.
Feel free to reopen it if needed.

Thank you!

**Pull request closed without merging**